README.md

```diff
@@ -44,6 +44,7 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [kolay-v/chessbot-reborn](https://github.com/kolay-v/chessbot-reborn) - The inline PvP chess game bot. [ᴜsᴇ](https://t.me/chess101bot)
 - [@OmniDubBot](https://t.me/OmniDubBot) - A bot that dubs your voice notes into 10+ languages.
 - [gabeklavans/gamejay-bot](https://github.com/gabeklavans/gamejay-bot) - Serves various multiplayer mini-games, inspired by GamePigeon for iOS. [ᴜsᴇ](https://t.me/gamejaybot)
+- [beilunyang/img-mom](https://github.com/beilunyang/img-mom) - A bot that uses telegram as cloud storage for images and can quickly upload images to cloud storage services like Cloudflare R2 and BlackBlaze B2. [ᴜsᴇ](https://t.me/img_mom_bot)
```

Comment:
I am unsure if "uses telegram as cloud storage" is ToS-friendly, aren’t you, too?

Reply:
I'm not sure. 
But the bot API does provide the ability to upload files as well as download them, and only specifies a maximum upload size of 20MB. 
Even if you use this bot, it won't bypass the 20MB limit.